db/post_migrate/20230522141920_add_bypass_invite_limits_permission.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AddBypassInviteLimitsPermission < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  class UserRole < ApplicationRecord; end
+
+  def change
+    admin_role     = UserRole.find_by(name: 'Admin')
+    moderator_role = UserRole.find_by(name: 'Moderator')
+
+    if admin_role
+      admin_role.permissions |= ::UserRole::FLAGS[:bypass_invite_limits]
+      admin_role.save
+    end
+
+    return unless moderator_role
+
+    moderator_role.permissions |= ::UserRole::FLAGS[:bypass_invite_limits]
+    moderator_role.save
+  end
```

Comment:
This migration would fail should `bypass_invite_limits` be removed at any future point.

```suggestion
  class UserRole < ApplicationRecord
    FLAGS = {
      bypass_invite_limits: (1 << 20),
    }
   end

  def change
    admin_role     = UserRole.find_by(name: 'Admin')
    moderator_role = UserRole.find_by(name: 'Moderator')

    if admin_role
      admin_role.permissions |= UserRole::FLAGS[:bypass_invite_limits]
      admin_role.save
    end

    return unless moderator_role

    moderator_role.permissions |= UserRole::FLAGS[:bypass_invite_limits]
    moderator_role.save
  end
```